usaspending_api/database_scripts/job_archive/backfill_unique_award_key.py

```diff
@@ -0,0 +1,172 @@
+import math
+import psycopg2
+import time
+
+from multiprocessing import Pool
+from os import environ
+from threading import Lock
+
+
+# DEFINE THIS ENVIRONMENT VARIABLE BEFORE RUNNING!
+# Obvs, this is the connection string to the database.
```

Comment:
Since I assume this is a situational script, can we also get comments explaining the circumstances this script was made under, when it should be used in the future, and what (if any) plans exist for sunset? This is how we got into the confusion with the modified agency list

Reply:
Good catch.  Added.